.github/workflows/update-product-schema.yml

```diff
@@ -0,0 +1,40 @@
+name: Update Product Schema
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 12 * * 1'
+
+env:
+  GIT_AUTHOR_EMAIL: 94923726+googleforcreators-bot@users.noreply.github.com
+  GIT_AUTHOR_NAME: googleforcreators-bot
+  GIT_COMMITTER_EMAIL: 94923726+googleforcreators-bot@users.noreply.github.com
+  GIT_COMMITTER_NAME: googleforcreators-bot
```

Comment:
Should these be hardcoded or should they be setup as part of the project's env vars?

Reply:
Great question. They are hardcoded like this in all of our GitHub Actions. This is just copy-pasted from other workflows. Don't currently see the need for moving these to env vars, but maybe I can do that in the future if we feel like it would make things easier.